src/plugin.ts

```diff
@@ -334,7 +353,10 @@ export function openGraphImagePlugin(options: Options): Plugin {
 
     cache.set(route.id, {
       routeLastModifiedAt,
-      imagePaths: images.map((image) => image.path),
+      images: images.map((image) => ({
+        name: image.name,
+        outputPath: image.path,
```

Comment:
Store the original destination path of the image in cache. This way, the plugin can copy the image from cache to this path when applicable.